code/datums/wires/wires.dm

```diff
@@ -194,7 +194,7 @@ GLOBAL_LIST_INIT(wireColours, list("red", "blue", "green", "black", "orange", "b
 	return 1
 
 /datum/wires/CanUseTopic(mob/user, datum/topic_state/state)
-	if(!CanUse(user))
+	if(holder && !CanUse(user))
```

Comment:
Would make more sense here to do this:
`if(!holder || !CanUse(user))`
That way you don't have to check for holder on line 144 either since the UI will be closed before it hits that line